src/main/java/ddingdong/ddingdongBE/file/service/S3FileService.java

```diff
@@ -0,0 +1,58 @@
+package ddingdong.ddingdongBE.file.service;
+
+import com.amazonaws.AmazonClientException;
+import com.amazonaws.AmazonServiceException;
+import com.amazonaws.HttpMethod;
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.s3.model.GeneratePresignedUrlRequest;
+import ddingdong.ddingdongBE.common.exception.AwsException.AwsClient;
+import ddingdong.ddingdongBE.common.exception.AwsException.AwsService;
+import java.net.URL;
+import java.util.Date;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class S3FileService {
+
+    @Value("${spring.s3.bucket}")
+    private String bucketName;
+
+    @Value("${spring.config.activate.on-profile}")
+    private String serverProfile;
+
+    private final AmazonS3Client amazonS3Client;
+
+    public URL generatePreSignedUrl(String path) {
+        path = createFilePath(path);
```

Comment:
변수를 새로 만드는 게 더 좋다고 생각하는데, 어떤가요?!
```suggestion
        String newName = createFilePath(path);
```